woocommerce/woocommerce
# Update e2e-utils, e2e-core-tests to merchant & shopper

### All Submissions:


### Changes proposed in this Pull Request:

This PR deprecates `StoreOwnerFlow` and `CustomerFlow` in favour of `merchant` and `shopper`. It also adds `gotoMyAccount()` to `shopper` to go to the `My Account` page.

Closes #27287 .

### How to test the changes in this Pull Request:

1. Run `npm install`
2. Edit a merchant test to use `StoreOwnerFlow`. Eg.
```
const { merchant, StoreOwnerFlow } = require( '@woocommerce/e2e-utils' );

const runActivationTest = () => {
       describe('Store owner can login and make sure WooCommerce is activated', () => {
               beforeAll(async () => {
                       await StoreOwnerFlow.login();
               });
```
3. Edit a shopper test to use `CustomerFlow`
4. Run `npm run build:packages`
5. Run E2E tests
6. A warning should be logged for each `StoreOwnerFlow` and `CustomerFlow` use
```
console.warn
CustomerFlow is deprecated. Please use shopper instead.

14 |
15 | const CustomerFlowDeprecated = () => {
> 16 | 	deprecated( 'CustomerFlow', {
| 	^
17 | 		alternative: 'shopper',
18 | 	});
19 | };
```

```
console.warn
StoreOwnerFlow is deprecated. Please use merchant instead.

20 |
21 | const StoreOwnerFlowDeprecated = () => {
> 22 | 	deprecated( 'StoreOwnerFlow', {
| 	^
23 | 		alternative: 'merchant',
24 | 	});
25 | };
```
7. The warnings should not cause the tests to fail

### Other information:


### Changelog entry

N/A
